### PR TITLE
chore: improve the contributor documentation

### PR DESCRIPTION
**Pull request by caicancai**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,4 +10,5 @@ This is a pure rust project, so only `cargo` is needed.
 - `cargo fmt` to format the current code.
 - `cargo build` to compile the current package.
 - `cargo clippy` to catch common mistakes and improve code.
-- `cargo test` to run unit tests.
\ No newline at end of file
+- `cargo test` to run unit tests.
+- `cargo bench` to run benchmark tests.
```
Comment (caicancai) [inline CONTRIBUTING.md:14]: Complete the contributor documentation

Comment (ethe) [reply CONTRIBUTING.md:14]: Yes, we are building test cases now

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -10,4 +10,5 @@ This is a pure rust project, so only `cargo` is needed.
 - `cargo fmt` to format the current code.
```
Comment (ethe) [inline CONTRIBUTING.md:10]: maybe we need to give a hint about using "cargo +nightly fmt" here?

Comment (caicancai) [reply CONTRIBUTING.md:10]: done

**Review by ethe — APPROVED** [review]
it looks good to me

**Comment by ethe** [comment]
Thank you @caicancai 🚀！

**Pull request merged**